libs/features/georoutingv2/georoutingv2.js

```diff
@@ -178,7 +178,7 @@ function buildContent(currentPage, locale, geoData, locales) {
   const titleText = geo.length ? geo[0][currentPage.geo] : '';
   const title = createTag('h3', { lang, dir }, locale.title.replace('{{geo}}', titleText));
   const text = createTag('p', { class: 'locale-text', lang, dir }, locale.text);
-  const flagFile = getCodes(locale).length > 1 ? 'globe-grid.png' : `flag-${locale.geo}.svg`;
+  const flagFile = `flag-${locale.geo.replace('_', '-')}.svg`;
```

Comment:
Why remove the globe-grid support?

Reply:
My initial approach was to add the column to decide when to show global grid and have documented that in jira ticket too. However that needed update in all consuming teams georoutingV2 file, hence went with this approach. 
@vhargrave I have few questions that I wanted to discuss about the concern you have raised. I'll sync up with you to understand it better.

Reply:
Can we go with both?  Check for the global flag first, then fallback to 404 method?

Reply:
@chrischrischris thanks for the suggestion. It sounds good and I can look into that.

Reply:
I have updated the code to handle both globe flag from georouting file and it will fallback to 404 also.
I also removed flags from git as we are fetching them from sharepoint now
@chrischrischris @vhargrave